source/content/drupal-10.md

```diff
@@ -12,17 +12,39 @@ product: [composer]
 integration: [--]
 ---
 
-Drupal 10 is coming in December 2022! We've been busy conducting early testing on the Pantheon platform, and have developed a process for creating a new Drupal 10 site on our platform.
+Drupal 10 is coming in December 2022! This page will be updated frequently with the latest information and instructions for Drupal 10 on Pantheon.,
 
-<Alert title="Warning" type="danger" >
+## Current Status of Drupal 10 at Pantheon
 
-These instructions were developed using beta versions of Drupal 10, and are for educational purposes only.  This page will be updated as production versions become available.
+| Scenario | Status | Documentation |
+|---|---|---|
+| Create a Drupal 10 site on Pantheon | Available | [Create a Drupal 10 Site](/drupal-10#create-a-drupal-10-site)|
+| Upgrade a Drupal 9 site to Drupal 10 manually | Available | [Upgrade from Drupal 9 to Drupal 10](/drupal-10#upgrade-a-drupal-9-site-to-drupal-10) |
+| Create Drupal 10 site from Pantheon Dashboard | Planned availability January 2023 | |
+| Upgrade a Drupal 9 site to Drupal 10 using the [Terminus Conversion Tools plugin](https://github.com/pantheon-systems/terminus-conversion-tools-plugin) | Early Access | Use the following command: `terminus conversion:upgrade-d10` |
+
+
+## Update a Drupal 9 Site to Drupal 10
+
+To update from version 9 to 10:
+
+1. If you are not already using PHP 8.1, go to your dashboard and install the update to PHP 8.1.
+
+1. Run [Drupal's Upgrade Status](https://www.drupal.org/project/upgrade_status).
+
+1. Use [Composer to update](/guides/upgrade-drupal-8-ic-to-drupal-9#set-drupal-core-version) to Drupal 10.  See the [Drupal release-specific documentation](https://www.drupal.org/project/drupal/releases) for more details.
 
-</Alert>
 
-## Before You Begin
 
-This process requires that you are on PHP 8.1.  To upgrade:
+## Create a Drupal 10 Site
+
+<Alert title="Note" type="info" >
+
+These instructions were developed using release candidate versions of Drupal 10.  This page will be updated as production versions become available.
+
+</Alert>
+
+### Before You Begin
```

Comment:
The section that follows does not need the "Add the following line" step or the "Commit and push the change" step

Reply:
Note that the section I am referring to was not modified by this PR and is hidden by default on the GitHub diff